src/app-layout/index.tsx

```diff
@@ -261,6 +326,14 @@ const OldAppLayout = React.forwardRef(
       [setSplitPanelSize, onSplitPanelResize]
     );
 
+    const onDrawerSizeSet = useCallback(
```

Comment:
Some legacy code in this file uses `useCallback`, but I would not add it to any new code we write

Reply:
Is there an alternative? I was copying what Split Panel does just above this.

Reply:
Just a plain function: 

```
const onDrawerSizeSet = ({detail}) =>  {}
```